sample-notificator-app/src/main/resources/notification-template/email-update-template.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <title>email-update-template</title>
+</head>
+<body>
+<div id="email-body">
+  <div id="person">Dear Max Mustermann,</div>
+  <br>
+  <div id="content">this is an automated email to inform you about recent updates on your subscribed project
+    <div id="project-information">My awesome project (ABCDE):</div>
```

Comment:
```suggestion
    <span id="project-information">My awesome project (ABCDE):</span>
```
I think this should be inline to complete the sentence. `this is an automated email to inform you about recent updates on your subscribed project My awesome project (ABCDE):`